src/Scrollbar.js

```diff
@@ -195,7 +213,8 @@ var Scrollbar = createReactClass({
     this._mouseMoveTracker = new DOMMouseMoveTracker(
       this._onMouseMove,
       this._onMouseMoveEnd,
-      document.documentElement
+      document.documentElement,
+      true
```

Comment:
We should pass props.touchEnabled into Scrollbar and make this this.props.touchEnabled